### ISSUE TITLE
[Python] client - missing import in models

### ISSUE THREAD

**Issue by davidgiga1993** [swagger-api/swagger-codegen#10173]
##### Description

When generating a python client for an API which uses schemas with `allOf` the referenced model is not imported in the code.

Here is some sample output code (comments have been removed for readability)

```python
# swagger_client/models/attribute_dto.py
# coding: utf-8
import pprint
import re  # noqa: F401

import six


class AttributeDto(object):
    swagger_types = {
        'dto_type': 'str',
        'id': 'int',
        'template_id': 'int',
        'type': 'int',
        'weight': 'int',
        'name': 'str',
        'value': 'str',
        'updated': 'datetime',
        'required': 'bool',
        'multi_value': 'bool',
        'selections': 'list[str]',
        'org': 'MinimalOrganizationDto',
        'project': 'MinimalProjectDto'
    }

    # Import of `TypedDto` is missing at the top
   # It should have `from swagger_client.models import TypedDto` as import
    if hasattr(TypedDto, "swagger_types"):
        swagger_types.update(TypedDto.swagger_types)
```

##### Swagger-codegen version

`3.0.19`

##### Swagger declaration file content or url
Since it's a giant file here is the full definition:
https://pastebin.com/x85KPJtC

##### Command line used for generation

```
java -jar swagger-codegen-cli-3.0.19.jar generate -i openapi.json -l python-o clients/python
```

##### Steps to reproduce

1) Create the client
2) Try to use the client

##### Suggest a fix/enhancement

The imports should be automatically added

**Comment by spinenkoia** [comment]
I have the same thing in 3.0.20

**Comment by michaelkarrer81** [comment]
same for me at 3.0.20 installed via brew on macosx - any workarounds yet?

**Comment by spinenkoia** [comment]
@kenjones-cisco 
please note that the bug is quite critical

**Comment by mikhailantoshkin** [comment]
Same issue here in 3.0.20 via swaggerapi/swagger-codegen-cli docker image

**Comment by sarangsbabu367** [comment]
Issue is `critical`. We are not able to run codegen due to this missing imports.

**Comment by HugoMario** [comment]
going to check this, thanks for reporting.

**Comment by michaelkarrer81** [comment]
Just another observation: 
It seems that for any of the "allOf" definitons a  model file (and model) is created? 
Is this the intended behaviour?

**Comment by HugoMario** [comment]
hey guys, i checked this issue and it looks fixed on [3.0.21-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/io/swagger/codegen/v3/swagger-codegen-cli/3.0.21-SNAPSHOT/swagger-codegen-cli-3.0.21-20200708.040844-57.jar) version. Can you please try with this jar? going to close it for now, but please, let me know if it's open and i work on it asap.
